GVRf/Framework/framework/src/main/java/org/gearvrf/GVRAssetLoader.java

```diff
@@ -696,6 +700,60 @@ public boolean stillWanted(GVRAndroidResource androidResource) {
         return sceneObject;
     }
 
+    static GVRModelSceneObject loadX3DModel(final GVRContext context,
+            String filePath, GVRResourceVolume.VolumeType volumeType,
+            EnumSet<GVRImportSettings> settings, boolean cacheEnabled)
+                    throws IOException {
+    	String fileName = filePath;
+    	File tmpFile = null;
+        AssetManager assetManager = context.getContext().getAssets();
+        GVRModelSceneObject root = new GVRModelSceneObject(context);
+        switch (volumeType) {
+        	case ANDROID_SDCARD:
+        	fileName = Environment.getExternalStorageDirectory().getAbsolutePath() + File.separator + filePath;
+        	break;
+
```

Comment:
Please fix the formatting. Maybe just run everything through the Eclipse formatter (the one for android).